credentials/credentials.ts

```diff
@@ -165,7 +161,25 @@ export class Credentials {
         this.accessTokenExpiryDate = new Date(Date.now() + response.data.expires_in * 1000);
       }
 
-      this.tokenCounter?.add(1, buildAttributes(response, this.authConfig));
+      if (this.telemetryConfig?.metrics?.counterCredentialsRequest?.attributes) {
+
+        let attributes = {};
+
+        attributes = TelemetryAttributes.fromRequest({
+          credentials: clientCredentials,
+          // resendCount: 0, // TODO: implement resend count tracking, not available in the current context
```

Comment:
@evansims can you expand on this TODO comment a bit?

Reply:
I just couldn't identify a readily accessible approach to bubbling the retry count down to that point without some significantly refactoring, but that might have changed with your updates

Reply:
I wrapped the response with retry count here, added to both credentials and API requests: https://github.com/openfga/js-sdk/pull/149/commits/1112a2c8ff26512b163cafb5f006bf93b96c5cf4